akka-docs/src/main/paradox/stream/operators/Broadcast.md

```diff
@@ -4,10 +4,32 @@ Emit each incoming element each of `n` outputs.
 
 @ref[Fan-out operators](index.md#fan-out-operators)
 
+## Signature
+
+@apidoc[akka.stream.*.Broadcast]
+
 ## Description
 
 Emit each incoming element each of `n` outputs.
 
+## Example
+
```

Comment:
Some friendly text explaining the sample would be nice. Perhaps also mention the fact that the downstreams are not running in parallel but on the same island/actor.

Something about the scala Tuple3 in the Java API would also be good.

Reply:
Added intro text.

"not running in parallel" was already mentioned, and there is even a separate example illustrating that

Tuple3 is not Scala. It's `import akka.japi.tuple.Tuple3`, which is shown in the included imports